starknet-foundry/crates/cast/tests/e2e/multicall_new.rs

```diff
@@ -0,0 +1,91 @@
+use crate::helpers::fixtures::default_cli_args;
+use crate::helpers::runner::runner;
+use cast::helpers::constants::DEFAULT_MULTICALL_CONTENTS;
+use tempfile::tempdir;
+
+#[tokio::test]
+async fn test_happy_case_stdout() {
+    let mut args = default_cli_args();
+
+    args.append(&mut vec!["multicall", "new"]);
+
+    let snapbox = runner(&args);
+    let bdg = snapbox.assert();
+    let out = bdg.get_output();
+
+    let stdout_str =
+        std::str::from_utf8(&out.stdout).expect("failed to convert command output to string");
+
+    assert!(out.stderr.is_empty());
+    assert!(stdout_str.contains(DEFAULT_MULTICALL_CONTENTS));
+}
+
+#[tokio::test]
+async fn test_happy_case_file() {
+    let mut args = default_cli_args();
+
+    let tmp_dir = tempdir().expect("failed to create temporary directory");
+    let tmp_path = tmp_dir.path().join("multicall.toml");
+    let tmp_path = tmp_path.to_str().expect("failed to convert path to string");
+
+    args.append(&mut vec!["multicall", "new", "--output-path", tmp_path]);
+
+    let snapbox = runner(&args);
+    let bdg = snapbox.assert();
+    let out = bdg.get_output();
+
+    let contents =
+        std::fs::read_to_string(tmp_path).expect("Should have been able to read the file");
+    assert!(out.stderr.is_empty());
+    assert!(out.stdout.is_empty());
```

Comment:
Shouldn't there be `"Multicall template successfully saved in {}"` in the output?

Reply:
yes [fix tests](https://github.com/foundry-rs/starknet-foundry/pull/98/commits/05633b26cebf494a8a28028e8380c0f84e554710)